### PR TITLE
feat: new MIME types and increase size

### PR DESCRIPTION
**Pull request by AntoineAugusti**
Bump max size from 2MB to 10MB and add additional allowed MIME types

Trello card: https://trello.com/c/rFIqFtVq/241-allow-extra-mime-types-for-attachments

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -1,15 +1,24 @@
 on: push
```
Comment (AntoineAugusti) [inline .github/workflows/push.yml:1]: Housekeeping as well, we don't update this repository often so doing similar changes that we did on the admin/API

File: app/config.py [hunk 2]
```diff
@@ -17,9 +17,16 @@ class Config(metaclass=MetaFlaskEnv):
...
         'text/csv',
         'text/plain',
+        'application/msword',  # .doc
+        'application/vnd.openxmlformats-officedocument.wordprocessingml.document',  # .docx
+        'image/jpeg',
+        'image/png',
+        'application/vnd.ms-excel',  # .xls
+        'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',  # .xlsx
+        'application/vnd.apple.numbers',  # "Numbers" app on macOS
     ]
```
Comment (AntoineAugusti) [inline app/config.py:27]: This is the real deal. Here are the new MIME types.

File: app/config.py [hunk 3]
```diff
@@ -17,9 +17,16 @@ class Config(metaclass=MetaFlaskEnv):
...
 
-    MAX_CONTENT_LENGTH = 2 * 1024 * 1024 + 1024
+    MAX_CONTENT_LENGTH = 10 * 1024 * 1024 + 1024
```
Comment (AntoineAugusti) [inline app/config.py:29]: Bumping from 2MB to 10MB

File: .github/workflows/docker.yaml [hunk 4]
```diff
@@ -6,8 +6,8 @@ on:
 
 env:
```
Comment (AntoineAugusti) [inline .github/workflows/docker.yaml:7]: Housekeeping, not the goal of this PR

**Review by willeybryan — APPROVED** [review]
I read the one file I understood

File: tests/upload/test_views.py [hunk 5]
```diff
@@ -100,7 +100,7 @@ def test_document_upload_unknown_type(client):
 
     assert response.status_code == 400
     assert response.json == {
-        'error': "Unsupported document type 'application/octet-stream'. Supported types are: ['application/pdf', 'text/csv', 'text/plain']" # noqa
+        'error': "Unsupported document type 'application/octet-stream'. Supported types are: ['application/pdf', 'text/csv', 'text/plain', 'application/msword', 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'image/jpeg', 'image/png', 'application/vnd.ms-excel', 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet', 'application/vnd.apple.numbers']" # noqa
```
Comment (jimleroyer) [inline tests/upload/test_views.py:103]: It would be nice to simply reuse `ALLOWED_MIME_TYPES` in the test so we don't have to update the same list..

**Review by jimleroyer — APPROVED** [review]
LGTM -- minor improvement possible but unrelated to the PR's goal.

File: tests/upload/test_views.py (hunk 5 above)
Comment (AntoineAugusti) [reply tests/upload/test_views.py:103]: Thought about that but I guess it defeats the purpose of the test if we reuse the same constant. Leaving it as-is for now.

Comment (jimleroyer) [reply tests/upload/test_views.py:103]: Testing a constant is testing a language feature. The way I see it, from the test name, it is meant to test uploads of unknown file types.

Comment (AntoineAugusti) [reply tests/upload/test_views.py:103]: I'm okay to test constants but if I change the constant (by adding/removing a MIME type) and I don't need to update tests, I think this is a code smell.

Comment (jimleroyer) [reply tests/upload/test_views.py:103]: That is a fair point. It feels like we are testing 2 different things though (behavior on unsupported types + the supported types). Ideally, we should have dedicated test(s) then for these.

**Pull request merged**